src/dropdown/dropdown.spec.ts

```diff
@@ -274,6 +274,28 @@ describe('ngb-dropdown-toggle', () => {
          expect(dropdownEl).toHaveCssClass('open');
        });
      })));
+  it('should have css class dropdown on div and dropdown-toggle on button',
```

Comment:
Correct me if I am wrong. 

`expect(buttonEl).toHaveCssClass('dropdown-toggle');` 

The `dropdown-toggle` class is also set via host. Ideally there should be a test case right?

This test is not there, am I missing out anything?